apps/etterlatte-hendelser-samordning/.run/etterlatte-hendelser-samordning.dev-gcp.run.xml

```diff
@@ -0,0 +1,34 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="etterlatte-hendelser-samordning.dev-gcp" type="JetRunConfigurationType">
+    <envs>
+      <env name="HTTP_PORT" value="8092" />
```

Comment:
Dette er samme port som grunnlag, sikkert lurt å ta en annen selv om det kanskje ikke er så ofte man kjører disse to samtidig

Reply:
Godt poeng